WebAPI/bookmark-manager.API/appsettings.Development.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": ""
+    "DefaultConnection": "Server=bookmark-manager-server.database.windows.net;Database=bookmark-manager;User ID=bookmark-administrator;Password=`E2wB6Au}H#F>%D:;Trusted_Connection=False;Encrypt=True;"
```

Comment:
Don't commit it
Now I have to change the password :)

Reply:
sorry, I forgot about it :)